### ISSUE TITLE
New ideas

### ISSUE THREAD

**Issue by sfaruque6** [AlexModGuy/Ice-and-Fire-Suggestions#786]
Here are some new mob ideas: pegasus, unicorn, elf, dwarf, demon, angel

Include creatures from other mythologies, such as Egyptian, Japanese, Abrahamic, Persian, Hindu, etc.

I think when you add vampires, add their own castles made of obsidian. These castles can only be found in extreme hills.

Pixies should also be in the flower forest biome

**Comment by Comdec35000** [comment]
Some mob ideas for these mythologies : 
- Egyptian : mummy (classic), beetle, anubis, Rê (falcon god of sun), Anubis (funeral rites god...)..
- Japanese : qilin, oni, kitsune, kappas, Tengu (Karasu tengu more than big nosed tengu please), some of the folklorics Yo-kais... (I think japanese dragon are overrated)
- Persian : Manticore, gouhls, griffins....
- Abrahamic : Golems ! (not only just meet golem like "hey good morning bro" but the ability to create golems with a rune and some clay to create a titanic golem and not only poppies supplier)

Just one more for fun : Nordic mythology !
-> Walkiries, Hugin and Munin (Odin two crows, one is black and the other is white)...

And two new bosses : Fenrir the godeater and Jormungand, the one who surrounds the world
They are two son of loki

**Comment by Spinoyonyx** [comment]
> Some mob ideas for these mythologies :
> 
> * Egyptian : mummy (classic), beetle, anubis, Rê (falcon god of sun), Anubis (funeral rites god...)..
> * Japanese : qilin, oni, kitsune, kappas, Tengu (Karasu tengu more than big nosed tengu please), some of the folklorics Yo-kais... (I think japanese dragon are overrated)
> * Persian : Manticore, gouhls, griffins....
> * Abrahamic : Golems ! (not only just meet golem like "hey good morning bro" but the ability to create golems with a rune and some clay to create a titanic golem and not only poppies supplier)
> 
> Just one more for fun : Nordic mythology !
> -> Walkiries, Hugin and Munin (Odin two crows, one is black and the other is white)...
> 
> And two new bosses : Fenrir the godeater and Jormungand, the one who surrounds the world
> They are two son of loki

Dont forget about the chinese mythologies like the half-dragon that are made as stone statues to guard palaces or for luck(like the dragon lion, dragon turtle and of course the Qilin

Also for Japanese its not Qilin its Kirin
Qilin is in Mndarin while Kirin is in Japanese for the chinese unicorn or 'horse dragon'

**Comment by Comdec35000** [comment]
Yes, I was disturbed because Kirin (麒麟) is also the japanese word for giraffe
It should be also cool some mooving satues, like a dragon lion statues whitch, when collide, get alive and try to kill you, like a guardian of a temple
